ctk/common/src/main/kotlin/org/codice/compliance/utils/RestAssuredExtensions.kt

```diff
@@ -127,7 +127,9 @@ fun Node.isNotHidden(): Boolean {
     return !HIDDEN.equals(this.getAttribute(TYPE_LOWER), ignoreCase = true)
 }
 
-fun Node.hasNoAttributeWithNameAndValue(attributeName: String,
-                                                expectedValue: String): Boolean {
+fun Node.hasNoAttributeWithNameAndValue(
+    attributeName: String,
+    expectedValue: String
+): Boolean {
```

Comment:
If I look at this new format objectively it looks cleaner, but I'm not used to it so it looks wrong 😄